### PR TITLE
[jdbc] Checkstyle updates for the JDBC binding.

### PR DESCRIPTION
**Pull request by allanbank**

### CODE REVIEW

**Comment by cmccoy** [comment]
@kruthar - think you could check this out?

File: jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java [hunk 1]
```diff
@@ -102,185 +135,210 @@ public int hashCode() {
...
     }
-  
+  }
+
   /**
-   * Initialize the database connection and set it up for sending requests to the database.
-   * This must be called once per client.
-   * @throws  
+   * Initialize the database connection and set it up for sending requests to
+   * the database. This must be called once per client. @throws
```
Comment (risdenk) [inline jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java]: @throws should be on a new line? also with a descripton

File: jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java [hunk 2]
```diff
@@ -102,185 +135,210 @@ public int hashCode() {
...
+  /**
+   * For the given key, returns what shard contains data for this key.
+   *
+   * @param key
+   *          Data key to do operation on
+   * @return Shard index
+   */
+  private int getShardIndexByKey(String key) {
+    int ret = Math.abs(key.hashCode()) % conns.size();
+    // System.out.println(conns.size() + ": Shard instance for "+ key + " (hash
```
Comment (risdenk) [inline jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java]: remove unused commented out code?

**Comment by risdenk** [comment]
Overall looks ok to me. The short lines (80 characters) makes it really hard to read though. @kruthar brought this up in https://github.com/brianfrankcooper/YCSB/issues/267#issuecomment-155269470.

@kruthar should be able to review this PR in more detail.

**Comment by kruthar** [comment]
Refactorings look good, also pulled, built and tested a few things. good.

File: jdbc/src/main/java/com/yahoo/ycsb/db/package-info.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2014, Yahoo!, Inc. All rights reserved.
```
Comment (busbey) [inline jdbc/src/main/java/com/yahoo/ycsb/db/package-info.java]: is this year and attribution correct?

**Comment by kruthar** [comment]
@allanbank - do you have time to fix the remaining small issues and rebase? I'd like to get this merged as there are a few more issues to tackle in the jdbc binding that are waiting on this PR because of the refactorings. If you're strapped for time I could also take over the PR, no problem.

**Pull request closed without merging**